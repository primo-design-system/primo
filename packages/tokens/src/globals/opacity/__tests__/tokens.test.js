const opacityTokens = require('../tokens.json');

describe('opacity tokens', () => {
  test(': matches snapshot', () => {
    expect(opacityTokens).toMatchSnapshot();
  });
});
