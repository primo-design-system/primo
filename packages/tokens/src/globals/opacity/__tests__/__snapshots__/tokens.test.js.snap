// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`opacity tokens : matches snapshot 1`] = `
Object {
  "opacity": Object {
    "0": Object {
      "value": 0,
    },
    "100": Object {
      "value": 0.1,
    },
    "200": Object {
      "value": 0.2,
    },
    "300": Object {
      "value": 0.3,
    },
    "400": Object {
      "value": 0.4,
    },
    "500": Object {
      "value": 0.5,
    },
    "600": Object {
      "value": 0.6,
    },
    "700": Object {
      "value": 0.7,
    },
    "800": Object {
      "value": 0.8,
    },
    "900": Object {
      "value": 0.9,
    },
  },
}
`;
